quickstarts/router-ext/README.md

```diff
@@ -0,0 +1,253 @@
+## Red Hat Process Automation Manager SmartRouter Extension Example
+
+Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.9.x/quickstarts/router-ext)
+
+This quickstart is intend to be used with the [RHPAM SmartRouter](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/master/smartrouter) image.
+
+The SmartRouter extensions are used to extend the Smart Router functionalities to make it to adapt to your needs.
+This example provides a very simple custom Kie Container resolver based on its version and how you can add this
```

Comment:
"of the contained" what it does mean?

Reply:
Sorry, typo - "on the version of the container".

Reply:
updated to: 
"This example provides a very simple custom Kie Container resolver based on the Kie Container version and how you can add this
extension to a custom Smart Router image."

to avoid confusion between container image and kie container.